app/models/application_record.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def audited_user_id
+    user = ::Audited.store[:current_user].call if ::Audited.store[:current_user].respond_to?(:call)
```

Comment:
Looks like [audited fetches the audit user like so](https://github.com/collectiveidea/audited/blob/4ad3a9c2a7dfacedc52f9f0c2a47f7b57a73e26e/lib/audited/audit.rb#L179):

```rb
    def set_audit_user
      self.user ||= ::Audited.store[:audited_user] # from .as_user
      self.user ||= ::Audited.store[:current_user].try!(:call) # from Sweeper
      nil # prevent stopping callback chains
    end
```

It probably doesn't matter much, but how come we're inverting the order (and not just copying the code directly)?